packages/airnode-deployer/src/cli/index.ts

```diff
@@ -6,31 +6,33 @@ import isEmpty from 'lodash/isEmpty';
 import uniq from 'lodash/uniq';
 import { hideBin } from 'yargs/helpers';
 import { CloudProvider, version as getNodeVersion } from '@api3/airnode-node';
+import { logger as loggerUtils } from '@api3/airnode-utilities';
 import { deploy, removeWithReceipt, remove } from './commands';
 import * as logger from '../utils/logger';
 import { version as packageVersion } from '../../package.json';
 import { longArguments, printableArguments } from '../utils/cli';
 
 function drawHeader() {
-  console.log(
-    '  ___  _                      _      \n' +
-      ' / _ \\(_)                    | |     \n' +
-      '/ /_\\ \\_ _ __ _ __   ___   __| | ___ \n' +
-      "|  _  | | '__| '_ \\ / _ \\ / _` |/ _ \\\n" +
-      '| | | | | |  | | | | (_) | (_| |  __/\n' +
-      '\\_| |_/_|_|  |_| |_|\\___/ \\__,_|\\___|\n'
+  loggerUtils.log(
+    [
+      '  ___  _                      _      ',
+      ' / _ \\(_)                    | |     ',
+      '/ /_\\ \\_ _ __ _ __   ___   __| | ___ ',
+      "|  _  | | '__| '_ \\ / _ \\ / _` |/ _ \\",
+      '| | | | | |  | | | | (_) | (_| |  __/',
+      '\\_| |_/_|_|  |_| |_|\\___/ \\__,_|\\___|',
+      `          Airnode v${getNodeVersion()}`,
+      `        Deployer CLI v${packageVersion}`,
+    ].join('\n')
   );
-  console.log(`\n          Airnode v${getNodeVersion()}`);
-  console.log(`        Deployer CLI v${packageVersion}\n`);
 }
 
 async function runCommand(command: () => Promise<void>) {
   try {
     await command();
   } catch (err) {
-    console.error(err);
-    // eslint-disable-next-line functional/immutable-data
-    process.exitCode = 1;
+    loggerUtils.error(err);
+    process.exit(1);
```

Comment:
I think this was written like that for a reason I do not remember. Maybe @amarthadan does?

Reply:
Yes, `yargs` didn't play well with calling `process.exit()`, I instead had to set the exit code and let them call the `exit` method.

Reply:
I'll revert it :+1: